extensions/2.0/Khronos/KHR_audio_emitter/README.md

```diff
@@ -0,0 +1,371 @@
+- `positional` Positional audio emitters play audio at a position in the scene. The properties are defined in the `positional` object. Using sound cones, the orientation is `-Z` having the same emission direction as [`KHR_lights_punctual`](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_lights_punctual) and [glTF cameras](https://github.com/KhronosGroup/glTF-Tutorials/blob/master/gltfTutorial/gltfTutorial_016_Cameras.md).
+
+#### `gain`
+
+Unitless linear multiplier against audio source volume used for determining audio emitter loudness. If not specified, the audio emitter volume gain is `1.0`.
+
+This value is linear, a value of `0.0` is no volume, `0.5` is half volume, `1.0` is the original volume, `2.0` is double the volume, etc. The final volume of the audio is a combination of this value, the audio source's gain, and if the audio emitter is positional, the relative positions of the emitter and listener.
+
+#### `sources`
+
+An array of audio source indices used by the audio emitter. This array may be empty. If empty or not specified, this emitter can be used to define how audio should emit from a node, but not which audio source to play.
+
+#### `positional`
+
+A sub-JSON object containing the positional audio emitter properties. This may only be defined if `type` is set to `positional`.
+
+### Positional Audio Emitter Properties
+
+When the audio emitter type is set to `positional`, additional properties may be defined on the `positional` object.
+
+#### Property Summary
+
+|                    | Type     | Description                                                                                                         | Default value                       |
+| ------------------ | -------- | ------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
+| **coneInnerAngle** | `number` | The anglular diameter of a cone inside of which there will be no angular volume reduction.                          | 6.2831853... (τ or 2π rad, 360 deg) |
+| **coneOuterAngle** | `number` | The anglular diameter of a cone outside of which the volume will be reduced to a constant value of `coneOuterGain`. | 6.2831853... (τ or 2π rad, 360 deg) |
+| **coneOuterGain**  | `number` | The linear volume gain of the audio emitter set when outside the cone defined by the `coneOuterAngle` property.     | 0.0                                 |
+| **distanceModel**  | `string` | Specifies the distance model for the audio emitter.                                                                 | `"inverse"`                         |
+| **maxDistance**    | `number` | The maximum distance between the emitter and listener, beyond which the audio cannot be heard.                      | 0.0                                 |
```

Comment:
Consider naming this property `range`, for further consistency with `KHR_lights_punctual`? Or perhaps `distanceMaximum` for greater consistency with iridescence?

https://github.com/KhronosGroup/glTF/blob/main/specification/2.0/ObjectModel.adoc#58-khr_materials_iridescence

Reply:
`maxDistance` is currently named this way to clarify the type of range, to be unambiguous with `refDistance`. If `maxDistance` is renamed, should `refDistance` be renamed too?

Reply:
This one was also done to be consistent with the Web Audio API that this spec was inspired from. Wide acceptance of this API is my only hesitation in renaming these properties. 

maxDistance: https://developer.mozilla.org/en-US/docs/Web/API/PannerNode/maxDistance